python/metatensor-operations/metatensor/operations/contiguous.py

```diff
@@ -0,0 +1,91 @@
+from . import _dispatch
+from ._backend import (
+    Labels,
+    TensorBlock,
+    TensorMap,
+    torch_jit_is_scripting,
+    torch_jit_script,
+)
+
+def is_contiguous(tensor: TensorMap) -> bool:
+    r"""
+    Checks contiguity of values and gradients in  a :py:class:`TensorMap`
+    
+    :param A: the input :py:class:`TensorMap`.
+
+    :return: a boolean indicating (non-)contiguity of values in ``A``.
```

Comment:
In general better to be explicit
```suggestion
    :return: a boolean indicating (non-)contiguity of values in ``A``. Returns true if all values and gradients arrays in the input ``tensor`` are contiguous, false otherwise.
```